src/pages/Login/LoginForm.js

```diff
@@ -103,10 +104,12 @@ const LoginForm = ({
         const resCondition = {
           SUCCESS: function () {
             localStorage.setItem('token', token);
-            localStorage.setItem('email', email);
-            localStorage.setItem('userId', userId);
-            localStorage.setItem('firstName', firstName);
-            localStorage.setItem('lastName', lastName);
+            onSetLoginedUserInfo({
+              email: email,
+              userId: userId,
+              firstName: firstName,
+              lastName: lastName,
```

Comment:
```suggestion
              email,
              userId,
              firstName,
              lastName
```

key와 value가 같으면 단축속성명을 쓸 수 있습니다